### PR TITLE
Feature/season statistics/most&least accurate team

### PR DESCRIPTION
**Pull request by MylesNottingham**
This will add methods to find most and least accurate teams along with their helper methods and all respective tests.

All Rubocop errors corrected.

All RSpec passes.

Spec_harness approved.

Coverage 100%.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -13,9 +13,15 @@ def self.from_csv(files)
   end
 
   def initialize(files)
-    @games = (CSV.open files[:games], headers: true, header_converters: :symbol).map { |row| Game.new(row) }
-    @teams = (CSV.open files[:teams], headers: true, header_converters: :symbol).map { |row| Team.new(row) }
-    @game_teams = (CSV.open files[:game_teams], headers: true, header_converters: :symbol).map { |row| GameTeam.new(row) }
+    @games = (CSV.open files[:games], headers: true, header_converters: :symbol).map do |row|
```
Comment (garrettgregor) [inline lib/stat_tracker.rb:16]: All changed per rubocop uniformity

**Review by javenb022 — APPROVED** [review]
Garrett did the heavy lifting fixing this, looks all good!

File: spec/game_team_spec.rb [hunk 2]
```diff
@@ -1,31 +1,68 @@
...
+        hoa: "away",
+        result: "LOSS",
+        settled_in: "OT",
+        head_coach: "John Tortorella",
+        goals: "2",
+        shots: "8",
+        tackles: "44"
+      }
+    )
+    @game_team_2 = GameTeam.new(
```
Comment (garrettgregor) [inline spec/game_team_spec.rb:18]: very thorough! Thanks for adding this!

File: spec/game_team_spec.rb [hunk 3]
```diff
@@ -1,31 +1,68 @@
...
+      expect(@game_team_2.result).to eq("WIN")
+      expect(@game_team_2.settled_in).to eq("REG")
+      expect(@game_team_2.head_coach).to eq("Dave Cameron")
+      expect(@game_team_2.goals).to eq(1)
+      expect(@game_team_2.shots).to eq(8)
+      expect(@game_team_2.tackles).to eq(34)
+    end
+  end
+
+  describe "#home?" do
```
Comment (garrettgregor) [inline spec/game_team_spec.rb:59]: Helper test 💥

**Review by garrettgregor — APPROVED** [review]
This all looks great now! I edited a few lines to incorporate the new `home?` method into the original helper methods from feature/highest-lowest and edited the file to resolve merge conflicts

**Review by MatthewTLim — APPROVED** [review]
Looks awesome!

**Pull request merged**